src/components/ProductInfo.vue

```diff
@@ -0,0 +1,392 @@
+<template>
+  <div v-if="!empty && !loading">
+    <div data-test="product-gallery"
+         class="col-md-4 col-md-offset-1 col-sm-6 product-gallery">
+      <ProductGallery v-if="images"
+                      :productImages="images" />
+    </div>
+    <div data-test="product-description"
+         class="col-sm-6 product-description">
+      <div class="row">
+        <div class="col-sm-12">
+            <h1 data-test="pdp-product-title"
+                class="text-uppercase pdp-product-title">
+              {{currentProduct.name}}
```

Comment:
Unfortunately ESLint doesn't complain about it, but usually we leave whitespaces around the variable name: `{{ currentProduct.name }}`.